lib/reporting/identity_verification_report.rb

```diff
@@ -151,13 +151,13 @@ def query
         fields
             name
           , properties.user_id AS user_id
-        | filter properties.service_provider = %{issuer}
+        #{issuer.present? ? '| filter properties.service_provider = %{issuer}' : ''}
         | filter name in %{event_names}
         | filter (name = %{usps_enrollment_status_updated} and properties.event_properties.passed = 1)
                  or (name != %{usps_enrollment_status_updated})
-        | filter (name = %{gpo_verification_submitted} and properties.event_properties.success = 1)
+        | filter (name = %{gpo_verification_submitted} and properties.event_properties.success = 1 and !properties.event_properties.pending_in_person_enrollment and !properties.event_properties.fraud_check_failed)
```

Comment:
should we drop a comment in `analytics_events.rb` near this event? along the lines of "if changing these properties, please update identity_verification_report.rb` too maybe?

Reply:
Is there prior art for this kind of comment? I don't want to develop a new pattern if one exists. I didn't see anything but `anaytics_events.rb` is huge.

Reply:
I dug through yards tags and found `@see`. I threw up a PR with that included a reference to the query method here.